### ISSUE TITLE
Firefox: right-hand side of 'in' should be an object, got undefined

### ISSUE THREAD

**Issue by jgarplind** [pushy/pushy-sdk-web#11]
On Firefox (123), this error is thrown consistently:

> right-hand side of 'in' should be an object, got undefined

This appears to be the problematic code block:

```
        // Check for browser.storage.local support
        return webExtensionNamespace in self && 'storage' in self[webExtensionNamespace] && 'local' in self[webExtensionNamespace].storage;
```

This is _not_ a critical bug for us (we don't expect Firefox users in our production environment), but I still think it would make sense to address this.

**Comment by pushy** [comment]
Hi @jgarplind,
Thanks so much for bringing this issue to our attention!

We've squashed this bug which was introduced in version `1.0.13` in commit 3a110cb and released version `1.0.18` of the [Pushy Web SDK](https://https://pushy.me/docs/web-push/get-sdk) to `npm`.

Please give it a try, and let us know if it indeed resolves the issue. 🎉

**Comment by jgarplind** [comment]
Rolling out as we speak.

Will know by Monday if that resolves the issue or not.

If I don't reply here by then, it is most certainly fixed.

Thanks for a rapid fix!
